README.md

```diff
@@ -66,38 +66,50 @@ Setup Environment
 
 Supported **MACHINE** targets (officially tested by Foundries):
 
+* am62xx-evm
```

Comment:
Mentioning @mwasilew only because README says "officially tested"

I'm not reviewing the machine list

Reply:
> Mentioning @mwasilew only because README says "officially tested"
> 
> I'm not reviewing the machine list

@angolini Thanks, I took a list from our "stable machine list" of jobserv.
@mwasilew Does it look ok to you?

Reply:
Yeah, this one is ok to add.

Reply:
some of them were not tested by me, so 'officially tested' means just 'confirmed working at some point in time'.